rustls/tests/api.rs

```diff
@@ -2349,6 +2349,62 @@ fn server_complete_io_for_write() {
     }
 }
 
+#[derive(Default)]
+struct EofWriter<const N: usize> {
+    written: usize,
+}
+
+impl<const N: usize> std::io::Write for EofWriter<N> {
```

Comment:
nit picking I think this `EofWriter` should also be below the test that uses it.

Reply:
Done.